content/en/monitoring/notification-manager.md

```diff
@@ -245,9 +247,9 @@ spec:
 EOF
 ```
 
->WechatApiCorpId is the id of your Wechat Work. WechatApiAgentId is the id of app which sending message to user in your Wechat Work, wechatApiSecret is the secret of this app, you can get these two parameters in App Managerment of your Wechat Work. Note that any user, party or tag who wants to rerceive notifications must be in the allowed users list of this app.
+> WechatApiCorpId is the id of your Wechat Work. WechatApiAgentId is the id of app which sending message to user in your Wechat Work, wechatApiSecret is the secret of this app, you can get these two parameters in App Management of your Wechat Work. Note that any user, party or tag who wants to receive notifications must be in the allowed users list of this app.
```

Comment:
WechatApiAgentId is the id of app which sending message to user in your Wechat Work =>
WechatApiAgentId is the id of the app sending message to user in your Wechat Work